utils/runtime-fuzzer/src/tests.rs

```diff
@@ -19,15 +19,21 @@
 use crate::*;
 use proptest::prelude::*;
 
-const MAX_GEAR_CALLS_BYTES: usize = 30_000_000;
-const MIN_GEAR_CALLS_BYTES: usize = 25_000_000;
+const MIN_GEAR_CALLS_BYTES: usize = 350_000;
```

Comment:
btw, it would be nice to be able to get bad corpus from CI when proptest failed

Reply:
this will be done another way - by obtaining crash input file from CI